<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Life Game - *</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/pic/pics/Tilda_Icons_12ft_card.svg</normaloff>:/pic/pics/Tilda_Icons_12ft_card.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>269</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="DrawWidget" name="drawWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuColors">
     <property name="title">
      <string>Colors</string>
     </property>
     <widget class="QMenu" name="menuCells_colors">
      <property name="title">
       <string>Cells colors</string>
      </property>
      <addaction name="actionColorCellAlive"/>
      <addaction name="actionColorCellNAlive"/>
      <addaction name="actionColorCellDead"/>
      <addaction name="actionColorCellNDead"/>
     </widget>
     <addaction name="actionColorBorder"/>
     <addaction name="actionColorText"/>
     <addaction name="menuCells_colors"/>
    </widget>
    <widget class="QMenu" name="menuCell_size">
     <property name="title">
      <string>Cell size</string>
     </property>
     <addaction name="actionCellSizeUp"/>
     <addaction name="actionCellSizeDown"/>
    </widget>
    <addaction name="menuColors"/>
    <addaction name="menuCell_size"/>
    <addaction name="actionShowHideImpact"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStep"/>
    <addaction name="separator"/>
    <addaction name="actionToroidal"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFillState"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuGame"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileNew"/>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileSave"/>
   <addaction name="separator"/>
   <addaction name="actionFillState"/>
   <addaction name="actionClear"/>
   <addaction name="actionShowHideImpact"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionStep"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionColorBorder">
   <property name="text">
    <string>Border color</string>
   </property>
   <property name="statusTip">
    <string>Change the color of cells' border</string>
   </property>
  </action>
  <action name="actionColorText">
   <property name="text">
    <string>Text color</string>
   </property>
   <property name="statusTip">
    <string>Change the color of impact numbers</string>
   </property>
  </action>
  <action name="actionColorCellAlive">
   <property name="text">
    <string>Alive</string>
   </property>
   <property name="statusTip">
    <string>Change the color of alive cells</string>
   </property>
  </action>
  <action name="actionColorCellNAlive">
   <property name="text">
    <string>Newly alive</string>
   </property>
   <property name="statusTip">
    <string>Change the color of newly alive cells</string>
   </property>
  </action>
  <action name="actionColorCellDead">
   <property name="text">
    <string>Dead</string>
   </property>
   <property name="statusTip">
    <string>Change the color of dead cells</string>
   </property>
  </action>
  <action name="actionColorCellNDead">
   <property name="text">
    <string>Newly dead</string>
   </property>
   <property name="statusTip">
    <string>Change the color of newly dead cells</string>
   </property>
  </action>
  <action name="actionCellSizeUp">
   <property name="text">
    <string>Up</string>
   </property>
   <property name="toolTip">
    <string>Cell size up to 5 pixels</string>
   </property>
   <property name="statusTip">
    <string>Change cell's size up to 5 pixels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionCellSizeDown">
   <property name="text">
    <string>Down</string>
   </property>
   <property name="toolTip">
    <string>Cell size down to 5 pixels</string>
   </property>
   <property name="statusTip">
    <string>Change cell's size down to 5 pixels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionShowHideImpact">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="/rec" resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_17ked_opendoors.svg</normaloff>:/pic/pics/Tilda_Icons_17ked_opendoors.svg</iconset>
   </property>
   <property name="text">
    <string>Impact</string>
   </property>
   <property name="toolTip">
    <string>Show/hide impact text</string>
   </property>
   <property name="statusTip">
    <string>Show/hide impact text</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_40_IT_animation.svg</normaloff>:/pic/pics/Tilda_Icons_40_IT_animation.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="statusTip">
    <string>Start/stop auto steps</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_8yo_asana.svg</normaloff>:/pic/pics/Tilda_Icons_8yo_asana.svg</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="statusTip">
    <string>Step forward</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionToroidal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toroidal</string>
   </property>
   <property name="statusTip">
    <string>Enable/dsable toroidal mode (needs even rows amount)</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_13ph_tune.svg</normaloff>:/pic/pics/Tilda_Icons_13ph_tune.svg</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="statusTip">
    <string>Game settings, e.g. size of board, interval of autostep, e.t.c.</string>
   </property>
  </action>
  <action name="actionFillState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_6bs_cosmetics.svg</normaloff>:/pic/pics/Tilda_Icons_6bs_cosmetics.svg</iconset>
   </property>
   <property name="text">
    <string>Replace/XOR</string>
   </property>
   <property name="toolTip">
    <string>Replace/XOR</string>
   </property>
   <property name="statusTip">
    <string>Set fill mode (replace lets fill cells by alive state, XOR lets you switch this state)</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset theme="/rec" resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_14ht_soap.svg</normaloff>:/pic/pics/Tilda_Icons_14ht_soap.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear canvas</string>
   </property>
   <property name="statusTip">
    <string>Clead the board</string>
   </property>
  </action>
  <action name="actionFileNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_40_IT_branding.svg</normaloff>:/pic/pics/Tilda_Icons_40_IT_branding.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Cread new field</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_30_system_folder.svg</normaloff>:/pic/pics/Tilda_Icons_30_system_folder.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open field</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_30_system_save.svg</normaloff>:/pic/pics/Tilda_Icons_30_system_save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save field</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="statusTip">
    <string>Save field as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_10cw_window.svg</normaloff>:/pic/pics/Tilda_Icons_10cw_window.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close the app</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/pics/Tilda_Icons_40_IT_abtest.svg</normaloff>:/pic/pics/Tilda_Icons_40_IT_abtest.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header>drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
